modules/appnexusBidAdapter.js

```diff
@@ -1138,12 +1147,13 @@ function hideSASIframe(elementId) {
   }
 }
 
-function outstreamRender(bid) {
+function outstreamRender(bid, context) {
```

Comment:
Since this new `context` option needs support from individual adapters anyway,, it looks to me like you could reduce the scope of this change if you look for the document / context in here, without requiring it to be passed around as an argument. By that I mean look at `this.options.context` in here to see what document / window to use, leaving almost everything in core unchanged.

Reply:
I think I agree after giving it some more thought, if the idea is that the same renderer instance could be used on different documents. The only pushback I'll give on is the naming again - `context` sound a lot like `object`, while the arument is very well defined as a document. Same for `contextResolver`.